src/utils/schemas/tableCardContent.json

```diff
@@ -64,12 +64,13 @@
       "properties": {
         "label": { "type": "string" },
         "dataSourceId": { "type": "string" },
+        "width": { "type": "string" },
```

Comment:
In tandem with my other comment about a custom validator, have we thought about more narrow types to support specific formats of strings (or other types) within the json config?